### DISCUSSION TITLE
Global Leaderboard considering all Contributors

### DISCUSSION THREAD

**Discussion by olegcherev** [devactivity-app/devActivity-Community#24] [Ideas & Feature Requests]
Organize contributors based on their accrued XP (Experience Points) across all Open Source projects installed devActivity.
Calculate XP the importance of the repository (based stars count) and event counts. No additional XP should be considered for challenges or custom achievements.

**Comment by damonexsppfx-alt** [comment]
This is great

**Reply by olegcherev** [reply]
Thanks @damonexsppfx-alt — glad this resonates.

Today leaderboards are **per workspace / installed org**, not a global board across every public repo that uses the app. The idea here is still on the list: rank contributors by XP across installed open-source projects, weight by repo importance (e.g. stars), and keep challenge/custom achievement XP out of that ranking.

We’ll keep this thread as the place to follow that work. If you’d use it, a short note on *why* (recognition, hiring signal, community health) helps us prioritize.
